AUTHORS

```diff
@@ -1415,5 +1410,276 @@ Bradley Curran <brad@bradleycurran.com>
+mskec <martin.skec@gmail.com>
+Peter Dave Hello <hsu@peterdavehello.org>
+JongChan Choi <disjukr@naver.com>
+Yihong Wang <yh.wang@ibm.com>
+Ryan Kelly <ryan@rfk.id.au>
+Alexander O'Mara <me@alexomara.com>
+James, please <jamesplease2@gmail.com>
+Josh Ferge <josh@augur.io>
+sreepurnajasti <jsreepur@in.ibm.com>
+Bidisha Pyne <bidipyne@in.ibm.com>
+David D Lowe <daviddlowe.flimm@gmail.com>
+rmdm <rmdm@users.noreply.github.com>
+Dávid Szakállas <david.szakallas@gmail.com>
+JiaLi.Passion <JiaLi.Passion@gmail.com>
+Paul Bininda <paul@bininda.com>
+sreepurnajasti <sreepurna.jasti@gmail.com>
+Gautam Mittal <200mittalgautam@gmail.com>
+Jamen Marzonie <jamenmarz@gmail.com>
+Jacob Jones <jacobfilmjones@hotmail.com>
+Vladimir Trifonov <trifonov.dev@gmail.com>
+aniketshukla <asr563up@gmail.com>
+Tarun <tarun.batra00@gmail.com>
+realwakka <realwakka@gmail.com>
+Gergely Nemeth <mail@nemethgergely.com>
+Samuel Reed <samuel.trace.reed@gmail.com>
+Anshul Guleria <anshulguleria@ymail.com>
+Justin Beckwith <justin.beckwith@gmail.com>
+Scott McKenzie <slammayjammay@gmail.com>
+jklepatch <julien@julienklepatch.com>
+Dan Homola <dan.homola@hotmail.cz>
+Chris Young <chris.young@gotinder.com>
+cornholio <0@mcornholio.ru>
+Tamás Hódi <tamas.hodi@risingstack.com>
+DuanPengfei <2459714173@qq.com>
+Ruben Bridgewater <ruben@bridgewater.de>
+LAKSHMI SWETHA GOPIREDDY <lgopired@in.ibm.com>
+Rob Wu <rob@robwu.nl>
+Steven Winston <swinston100@hotmail.com>
+sallen450 <jqh101@sina.com>
+OriLev <sviatos@gmail.com>
+Zongmin Lei <leizongmin@gmail.com>
+lena <kaplanlena@gmail.com>
+Julien Klepatch <julien@julienklepatch.com>
+Azard <330815461@qq.com>
+Ezequiel Garcia <ezequiel@vanguardiasur.com.ar>
+Kyle Farnung <kfarnung@microsoft.com>
+starkwang <381152119@qq.com>
+Nataly Shrits <nataly@soluto.com>
+Jaime Bernardo <jaime@janeasystems.com>
+Natanael Log <natte.log@gmail.com>
+MoonBall <gangc.cxy@foxmail.com>
+kuroljov <viktor.konsta@gmail.com>
+Matt Sergeant <matt@sergeant.org>
+Eduardo Leggiero <esleggiero@gmail.com>
+Moogen Tian <moogentian@outlook.com>
+Jimmy Thomson <jithomso@microsoft.com>
+David Drysdale <drysdale@google.com>
+Roman Shoryn <romanshoryn@gmail.com>
+AJ Jordan <alex@strugee.net>
+Peter Czibik <p.czibik@gmail.com>
+章礼平 <LipperZack@gmail.com>
+Fraser Xu <xvfeng123@gmail.com>
+Song, Bintao Garfield <songbintaochina@gmail.com>
+Flandre <panyilinlove@qq.com>
+akira.xue <akira.xue@guanaitong.com>
+Bang Wu <ohwubang@gmail.com>
+kadoufall <spuspu1997@gmail.com>
+jiangplus <jiang.plus.times@gmail.com>
+tobewhatwewant <tobewhatwewant@gmail.com>
+blade254353074 <blade254353074@hotmail.com>
+weiyuanyue <millywey@qq.com>
+xinglong.wangwxl <hacke2@qq.com>
+vercent deng <vercent.deng.lqy@gmail.com>
+boydfd <boydfd@163.com>
+Superwoods <st_sister@iCloud.com>
+Weijia Wang <381152119@qq.com>
+shaman <wow2009zfx@gmail.com>
+Zhang Weijie <email@zhangweijie.cn>
+Gunar C. Gessner <gunar@gunargessner.com>
+SkyAo <csvwolf@qq.com>
+Benjamin Gruenbaum <benji@peer5.com>
```

Comment:
This is most likely a duplicate of inglor@gmail.com (work mail vs. old personal mail)